### ISSUE TITLE
Can't create `WarperCreator` and use `StitcherTrait::set_warper`

### ISSUE THREAD

**Issue by markhilb** [twistedfall/opencv-rust#557]
I am trying to stitch a panorama using the `Sitcher` struct with the warper: `TransverseMercatorWarper`.
However, the `StitcherTrait::set_warper` method takes in a `Ptr<WarperCreator>` which I can't seem to create,
since neither `TransverseMercatorWarper` nor `WarperCreator` has a `new`/`create` method.

Here is some example code showing the problem:
```rust
use opencv::{
    core::{Mat, Vector},
    imgcodecs::{imread_def, imwrite_def},
    stitching::{Stitcher, StitcherTrait, Stitcher_Mode},
    xfeatures2d::SURF,
};

pub fn stitch(files: Vec<String>) -> anyhow::Result<()> {
    let mut images = Vector::<Mat>::with_capacity(files.len());
    for f in files {
        images.push(imread_def(&f)?);
    }

    let mut stitcher = Stitcher::create(Stitcher_Mode::PANORAMA)?;

    let surf = SURF::create_def()?;
    stitcher.set_features_finder(surf.into())?;

    // What goes here?
    let warper = todo!();
    stitcher.set_warper(warper)?;

    let mut output = Mat::default();

    stitcher.stitch(&images, &mut output)?;
    imwrite_def("result.png", &output)?;
    Ok(())
}
```

In c++ the missing part would be written like this:
```c++
  auto warper = makePtr<cv::TransverseMercatorWarper>();
  stitcher->setWarper(warper);
```

Any idea how to create the warper in rust?

**Comment by twistedfall** [comment]
Looks like currently it's not possible to create an instance of the necessary warper. I'm going to look into this, thanks for the report!

**Comment by twistedfall** [comment]
Can you please check the `rel` branch to see it the following now works as expected:
```rust
    // What goes here?
    let warper = Ptr::new(TransverseMercatorWarper::default());
    stitcher.set_warper(warper.into())?;
```

**Comment by markhilb** [comment]
Yes, that works great!

**Comment by twistedfall** [comment]
Perfect, I will release the version with the fix shortly

**Comment by twistedfall** [comment]
This has been released as part of 0.90 version
